control-plane/subcommand/sync-catalog/command.go

```diff
@@ -138,6 +143,10 @@ func (c *Command) init() {
 			"\"debug\", \"info\", \"warn\", and \"error\".")
 	c.flags.BoolVar(&c.flagLogJSON, "log-json", false,
 		"Enable or disable JSON output format for logging.")
+	c.flags.StringVar(&c.flagPurgeK8SServicesFromNode, "purge-k8s-services-from-node", "",
```

Comment:
I'm wondering if this should be a `c.flags.BoolVar`, what kind of string are we expecting users to provide to purge k8s service? Or simply providing this flag will initiate a purge